gulpfile.js

```
@@ -549,9 +549,15 @@ gulp.task('build.tools', ['build/clean.tools'], function(done) {
 
 // private task to build tools
 gulp.task('!build.tools', function() {
+  var TSC_OPTS = {
+    target: 'ES5',
+    module: 'commonjs',
+    typescript: require('typescript'),
```

That's not the case actually. ts2dart must run with the version of typescript it was compiled with. Here, we can run with any 1.5 version of typescript. For the angular compile, we must run with bleeding edge typescript. So there is no requirement that any of these three different things use the same version.
It would be nice to fail faster in ts2dart, rather than claim that SyntaxKind.SourceFile is not supported. I'll look into that.